weave

```diff
@@ -1211,6 +1212,7 @@ launch_proxy() {
         --entrypoint=/home/weave/weaveproxy \
         $WEAVEPROXY_DOCKER_ARGS $EXEC_IMAGE $COVERAGE_ARGS $PROXY_ARGS)
     wait_for_log $PROXY_CONTAINER_NAME "proxy listening"
+    ln -sf -T /var/run/weave/weave.sock /var/run/weave.sock
```

Comment:
We could do, but it's not a huge win when it's being removed in the next version anyway.